Automattic/wp-calypso
# Pricing page rework: Add featured product cards

#### Proposed Changes

* Add assets to `client/assets/images/jetpack`
* Add `HeroImage` component
* Create reusable `FeaturedItemCard` component to use for both products and bundles
* Create `useCreateCheckout` hooks to keep components clean
* Enhance `DisplayPrice` component to allow a change in the order of original and discounted price
* Add `FeaturedItemCard` both in product and bundle list

#### Testing Instructions



* click on Jetpack Cloud live link below
    * Goto `/pricing?flags=jetpack/pricing-page-rework-v1`
* or boot up this PR 
    * Run `git fetch && git checkout add/jp-pricing-rework/featured-product-cards`
    * Run `yarn start-jetpack-cloud`
    * Goto [http://jetpack.cloud.localhost:3000/pricing?flags=jetpack/pricing-page-rework-v1](http://jetpack.cloud.localhost:3000/pricing?flags=jetpack/pricing-page-rework-v1)
* Confirm that you see "Backup" and "VideoPress" featured cards in "Most popular products"
* Goto Bundles view
* Confirm that you see "Security" and "Complete" featured cards under "Most popular bundles"
* Confirm that the design matches what is in Firma

#### Pre-merge Checklist






Completes: 1202796695664022-as-1202796695664102/f and 1202796695664022-as-1202796695664123/f

<img width="1430" alt="Screenshot 2022-08-29 at 5 38 46 PM" src="https://user-images.githubusercontent.com/18226415/187197954-b4d940f5-1abf-42cd-9772-40e57c018e57.png">
<img width="1386" alt="Screenshot 2022-08-29 at 5 38 55 PM" src="https://user-images.githubusercontent.com/18226415/187197980-236f98ae-926b-427f-a24b-dd88a735b90c.png">
<img width="428" alt="Screenshot 2022-08-29 at 5 39 43 PM" src="https://user-images.githubusercontent.com/18226415/187197989-228cf662-f3e9-4e68-9858-9ce61cf96641.png">
<img width="418" alt="Screenshot 2022-08-29 at 5 39 54 PM" src="https://user-images.githubusercontent.com/18226415/187197995-13429d3e-f772-4658-83bd-36eac2e5cf0c.png">


---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - 0-as-1202796695664123
  - 0-as-1202796695664102
